Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:12-alpine3.9
```

Comment:
Why not just latest-alpine?

Reply:
was thinking of LTS actually, though maybe it isnt necessary?

Reply:
I lean towards latest only because I'm not too worried about breakages with this (heaven forbid if you use this in production)

Reply:
latest-alpine apparently doesnt exist, gonna just use latest